src/content/docs/en/basics/layouts.mdx

```diff
@@ -62,18 +62,49 @@ import MySiteLayout from '../layouts/MySiteLayout.astro';
 </MySiteLayout>
 ```
 
-
 <ReadMore>Learn more about [slots](/en/basics/astro-components/#slots).</ReadMore>
 
+## Using TypeScript with layouts
```

Comment:
Just noting that I moved this up for prominence, and to push the Markdown layout content (more niche/specific use case) lower on the page.